app/src/js/components/Table/Table.js

```diff
@@ -92,6 +96,11 @@ class List extends React.Component {
       selected,
       clearSelected: false,
     });
+
+    // Current selection is passed to the parent component
+    if (typeof this.onSelect === 'function') {
+      this.onSelect(selected);
+    }
```

Comment:
Instead of setting this.onSelect in the constructor, we could just do this:

```
if (typeof this.props.onSelect === 'function') {
  this.props.onSelect(selected);
}
```

Prevents us having to check for a function twice

Alternatively, maybe we could do something like setting `this.onSelect` to `function(){}` in the constructor so we don't have to check if it's a function here 🤷

Reply:
Sure! Originally I was thinking we should not have this function reach into `this.props` but you're totally right, it saves having to validate twice. And I think doing it that way might be a little easier to read than defining a new `this.onselect`.